tests/scripts/insert_sessions_categories.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * This script generates four session categories.
+ */
+require_once '../../main/inc/global.inc.php';
+api_protect_admin_script();
+$categories = array(
```

Comment:
This is a short example script (we needed these categories names specifically). Given it is in tests/scripts/ I don't think the rules with content name really need to apply. The rest still does (license, security, formatting, variables names, etc). Just that the content is specific.